source/django-get-started/django-connect-mongodb.txt

```diff
@@ -0,0 +1,96 @@
+.. _django-get-started-connect:
+
+=================================
+Configure your MongoDB Connection
+=================================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: app, odm, code example
+
+After installing {+django-odm+} and creating a MongoDB Atlas deployment,
+you can create a Django project that connects to MongoDB.
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create a Django project
+
+      From your shell, run the following command to create a 
+      new Django project called ``quickstart`` based on a custom template:
+      
+      .. code-block:: bash
+
+         django-admin startproject quickstart --template https://github.com/mongodb-labs/django-mongodb-project/archive/refs/heads/{+django-version-number+}.x.zip
+      
+      .. note:: Project Template
+
+         The ``django-mongodb-project`` template resembles the default Django project
+         template but makes the following changes:
+         
+         - Includes MongoDB-specific migrations
+         - Modifies the ``settings.py`` file to instruct Django
+           to use an ``ObjectId`` value as each model's primary key
+
+      After running this command, your ``quickstart`` project has
+      the following file structure:
+
+      .. code-block:: bash
+         :copyable: false
+
+         quickstart/
+            manage.py
+            mongo_migrations/
+                __init__.py
+                contenttypes/
+                auth/
+                admin/
+            quickstart/
+                __init__.py
+                apps.py
+                settings.py
+                urls.py
+                asgi.py
+                wsgi.py
+
+   .. step:: Update your database settings
+
+      Open your ``settings.py`` file and navigate to the ``DATABASES`` setting.
+      Replace this setting with the following code:
+
+      .. code-block:: bash
+
+         DATABASES = {
+             "default": {
+                 "ENGINE": "django_mongodb_backend",
+                 "NAME": "sample_mflix",
+                 "USER": "<username>",
+                 "PASSWORD": "<password>",
+                 "HOST": "<connection string URI>",
+             },
+         }
+
+      Replace the ``<username>`` and ``<password>`` placeholders with your
+      Atlas database user's username and password. Then, replace the
+      ``<connection string URI>`` placeholder with the connection string
+      that you copied from the :ref:`django-get-started-connection-string`
+      step of this guide. This configures your Django app to connect to
+      your Atlas deployment and access the ``sample_mflix`` sample database.
+
+   .. step:: Start the server
+
+      To verify that you installed {+django-odm+} and correctly configured
+      your project, run the following command from your project root:
+
+      .. code-block:: bash
+
+         python3 manage.py runserver
```

Comment:
Maybe this is consistent with the rest of your documentation and you want to leave it alone, but if you are working in a Python 3 virtual environment, you can write `python` without the 3. Coming from Django's docs, it looks odd to include that 3 on all commands.

Reply:
I was going to raise this too, but would defer to MongoDB-wide conventions. If there are no such conventions, then I prefer `python` too.